tests/snippets/stdlib_os.py

```diff
@@ -415,4 +415,25 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     for arg in [None, 1, 1.0, TabError]:
         assert_raises(TypeError, os.system, arg)
 
+if "win" in sys.platform:
```

Comment:
Unfortunately, macos is `darwin` so it also inclues `win`. `sys.platform.startswith('win')` will work